frontend/src/polyfills.ts

```diff
@@ -58,6 +58,17 @@ import 'zone.js'; // Included with Angular CLI.
 // from https://github.com/valor-software/ng2-dragula/#1-important-add-the-following-line-to-your-polyfillsts
 (window as any).global = window;
 
+/***************************************************************************************************
+ * WORKFLOWS METHODS
+ */
+
+declare global {
+  interface Window {
+    __fulib_org_setIndexFromIframe;
+    __fulib_org_changeFrameWithToast;
```

Comment:
Implicitly typed as any, please define the type.

Reply:
I was thinking about this some more and actually I believe it is cleaner and safer to use [`postMessage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) together with [`@HostListener('window:message')`](https://angular.io/api/core/HostListener#usage-notes).

Also consider that your WorkflowsComponent currently leaks itself because it does not remove the two listeners in `ngOnDestroy`. `@HostListener` solves this automatically.

Reply:
I added and tested this locally. Currently it is still WIP because there needs to be a new fulibWorkflows Version so this really works.